Emitron/Emitron.xcodeproj/project.pbxproj

```diff
@@ -2588,7 +2588,7 @@
 				CURRENT_PROJECT_VERSION = UNKNOWN;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				DEVELOPMENT_ASSET_PATHS = "Emitron/Preview\\ Content";
-				DEVELOPMENT_TEAM = KFCNEC27GU;
+				DEVELOPMENT_TEAM = 6HHQWGY2HB;
```

Comment:
We shouldn't be changing the development team

Reply:
Reverted the development team